### DISCUSSION TITLE
Add "commit suggestion" button in PR Reviews on mobile devices (GitHub Mobile App)

### DISCUSSION THREAD

**Discussion by trueberryless** [community/community#141743] [Mobile]
The one and only thing missing in the GitHub Mobile App is the button for accepting suggestions from reviewers of your PR. I always have to open the PR in the web to quickly update the code with the suggestions which is unnecessary...

**Comment by MorelliniMatteo** [comment]
Feedback to GitHub
You might want to consider providing feedback directly to GitHub about this feature request. Here’s a sample message you could use:

Subject: Feature Request: Add "Accept Suggestions" Button for PRs in GitHub Mobile App

Body:

Hi GitHub Team,

I wanted to provide some feedback regarding the GitHub Mobile App. One feature that I find missing is the ability to accept suggestions from reviewers directly within the app while reviewing pull requests (PRs).

Currently, whenever I receive suggestions, I have to open the PR in a web browser to accept and implement these suggestions, which feels unnecessary and disrupts my workflow.

Having an "Accept Suggestions" button in the mobile app would streamline the process significantly and improve the overall user experience.

Thank you for considering this suggestion!

Best regards,
[Your Name]
[Your GitHub Username]

Workarounds While Waiting for Updates
While waiting for GitHub to potentially add this feature, here are some workarounds:

Use Desktop or Web Version:

Whenever possible, use a desktop or web version of GitHub to manage PRs, especially when needing to implement multiple suggestions quickly.
Leave Comments:

If you’re unable to accept suggestions through the mobile app, consider leaving comments on specific lines in the code as a temporary measure. This way, you can note what changes you intend to make before revisiting the PR in the web app.
Set Up Notifications:

Ensure that your notifications are set up correctly so that you can stay updated on PR suggestions and comments, helping you manage changes more effectively.
Third-Party Apps:

Explore third-party apps that may offer more comprehensive PR management features on mobile, although this would involve a trade-off in terms of integration with GitHub.
Feature Requests on GitHub Discussions:

Consider engaging with the GitHub Community Discussions page or similar forums to gauge interest in this feature and gather support from other users. The more feedback they receive, the more likely they are to prioritize such features.
